pkg/ptrace/ptrace_nocgo.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+//go:build !cgo
```

Comment:
This is duplicated with `+build !cgo`?

Reply:
Yes, I will replace them.

Reply:
Updated, except `// +build !ignore_autogenerated` in `zz_generated.deepcopy.go`, which is controlled by controller-gen.